### ISSUE TITLE
Production readiness

### ISSUE THREAD

**Issue by michalSolarz** [oskardudycz/GoldenEye#89]
@oskardudycz Hi, do you think that GoldenEye is safe for production use cases or should we wait for stable 1.0 realase? If not do you know some production ready alternative (I know about eventflow but 1.0 version wasn't released yet)?

**Comment by oskardudycz** [comment]
@michalSolarz, it's safe in a way that it worked on my production and works still on a few others. However, I'm not running it at that moment on production, so I don't have a chance to dogfood it. Currently, it's more my pet project rather than framework, I advocate.

I started the cleanup to make it simpler and cleaner. I plan to migrate my ideas from [Event Sourcing samples](https://github.com/oskardudycz/EventSourcing.NetCore) here when I find some time. I started to migrate it to .NET 6 https://github.com/oskardudycz/GoldenEye/pull/90.

TLDR at this point, as I'm not getting money for that, I cannot guarantee the response times and deliveries. The framework works as is, and it's a decent tool. I think that if you like its approach, then you can use NuGet or fork it.

If you'd like to start contributing, I would be happy to take some pull requests with the initial discussion on the scope.
